asset_pipeline/chunked_upload.py

```diff
@@ -27,12 +27,15 @@ def upload_chunked_file(self, base_url=None, file_path=None, early_return_on_err
             raise Exception('Could not upload file.')
         return response.json()['file_url']
 
-    def _chunked_upload_file(self, base_url=None, file_path=None, early_return_on_error=True, md5=None):
+    def _chunked_upload_file(self, base_url=None, file_path=None, early_return_on_error=True, md5=None, chunk_size_in_bytes=None):
```

Comment:
You could do this (add in the docstring that the default is 2 MiB)
```suggestion
    def _chunked_upload_file(self, base_url=None, file_path=None, early_return_on_error=True, md5=None, chunk_size_in_bytes=2<<20):
```